mercury.spec

```diff
@@ -13,7 +13,7 @@ Group:		Development/Libraries
 License:	Aregonee National Laboratory, Department of Energy License
 URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
-Patch0:		https://github.com/daos-stack/mercury/daos_9173_workaround.patch
+Patch0:		https://github.com/daos-stack/mercury/cpu_usage.patch
```

Comment:
The `daos_9173_workaround.patch` should have been removed in this PR.

Reply:
we want to keep this daos_9173 patch around for now in case we have to revert back to it